README.md

```diff
@@ -5,7 +5,7 @@
 Materia (formerly Flat-Plat) is a [Material Design](https://material.io) theme for GNOME/GTK+ based desktop environments.  
 It supports GTK+ 3, GTK+ 2, GNOME Shell, Budgie, Cinnamon, MATE, Unity, LightDM, GDM, Chrome theme, etc.
 
-Also Materia has compatibility with [oomox theme designer](https://github.com/themix-project/oomox).
+Also Materia can be customized in [oomox theme designer](https://github.com/themix-project/oomox).
```

Comment:
my english is not so good so i am actually not sure if it should be `in`, `with` or `using`

Reply:
My English is also not so good. :)

but I think `with` or `using` is correct (or better) in this case.

Can you update this to use `with`?